docker/safety-rules/Dockerfile

```diff
@@ -24,7 +24,8 @@ FROM debian:buster AS prod
 
```

Comment:
Why are we building libra-node in this Dockerfile? (Referring to line 19)

Do we expect safety rules and libra-node to be in the same image?

Reply:
I think the intent is to build everything, everywhere. But I don't personally know.

Reply:
Looks like we're discarding the `libra-node` binary built here, can we remove ` -p libra-node` from here so that builds can be faster?

Reply:
Let's not bring in more complexity to this PR.